products/kms/api.yaml

```diff
@@ -65,7 +65,7 @@ objects:
   - !ruby/object:Api::Resource
     name: 'CryptoKey'
     base_url: '{{key_ring}}/cryptoKeys'
-    create_url: '{{key_ring}}/cryptoKeys?cryptoKeyId={{name}}'
+    create_url: '{{key_ring}}/cryptoKeys?cryptoKeyId={{name}}&skipInitialVersionCreation={{skipInitialVersionCreation}}'
```

Comment:
I think you want the part in `{{}}` to be `snake_case`, like `{{skip_initial_version_creation}}`.

Reply:
fixed